currency/pair.go

```diff
@@ -10,6 +10,9 @@ import (
 // a Pair struct
 func NewPairDelimiter(currencyPair, delimiter string) Pair {
 	result := strings.Split(currencyPair, delimiter)
+	if len(result) > 2 {
+		result[1] = strings.Join(result[1:], delimiter)
+	}
 	return Pair{
 		Delimiter: delimiter,
 		Base:      NewCode(result[0]),
```

Comment:
This is something that also happens in master if you pass a incorrectly delimited pair into NewPairDelimiter()

E.g

```go
NewPairDelimiter("BTC-MOVE-0626", "/")
```

It will cause a out of range panic on the return